blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: blog
+root: ../../..
+author: Victor (Kwnagchun) Lee
+title: "Korean Translation of Software Carpentry - version 5.2"
+date: 2015-04-16
+time: "13:00:00"
+category: ["Translation", "Version 5.2", "Korea", "Book"]
+---
+<!-- start excerpt -->
+<p> Based on Software Carpentry version 5.2, we translated <a href="http://www.xwmooc.net/swc/">all the lessons</a> in the form of pdf, html, mobi, epub, and azw3. It took almost 6 months: the first half spent on mostly lessons, and the second half spent on remainings. While translating and building various ebook formats, the original book production software of Software Carpentry seemed not to have upfront architectural concerns in terms of CJK problems. Also, we are wondering how to generate various book formats from the current developing lessons. 
```

Comment:
Can you replace "CJK problems" with "problems due  Chinese, Japanese, and Korean characters". I needed to search on Wikipedia for CJK.